### PR TITLE
Readium sdk for lcp

### PR DESCRIPTION
**Pull request by drminside**
I send a pull request for Readium SDK to implement LCP whose target is develop.

However it shows "can't automatically merge" message, which had no at the request to the master.

### CODE REVIEW

**Comment by bluefirepatrick** [comment]
@drminside we are going to look at merging this pull request this week.  We will ping you if there are any difficulties.

**Comment by nleme** [comment]
I'm taking a look at this pull request. I was able to download the code from your branch (drminside:feature/sdkForLcp) and build it in Xcode. I was able to run the Launcher with it. So far it worked fine, I was able to open "Children's literature" and "Shared Culture" and play the video in it without any problem. I'll be taking a deeper look at the code now.

**Comment by danielweck** [comment]
By the way, this suersedes https://github.com/readium/readium-sdk/pull/211

**Comment by nleme** [comment]
@drminside 

So, after some discussion around this pull request in the latest Readium SDK meeting, we are all in to get this pull request merged into develop.

I took a look to see if I could do that, however currently GitHub is stating that there are merge conflicts in the code, and therefore the pull request cannot be done.

I was trying to make a manual merge of the pull request, using the command line tools, but I ended up giving up on that. I was concerned that I could mess up the merge, and on top of that I don't have the LCP code that you guys are using to make sure that the change is still working fine with LCP.

Given that, can you guys solve the merge conflicts? If the command line tools become too difficult for you guys, you always have the alternative of creating a branch from develop and then copy your changes there, and that for sure would work. Then, once the merge conflicts are solved, you can just do a merge automatically. Anyways, let me know how that goes with you.

File: ePub3/ePub/encryption.cpp [hunk 1]
```diff
@@ -45,6 +52,32 @@ bool EncryptionInfo::ParseXML(shared_ptr<xml::Node> node)
...
+    if (!strings.empty())
+    {
+        for ( int i = 0 ; i < strings[0].size() ; i++)
+            if (strings[0][i] < '0' || strings[0][i] > '9' )
+                return false;
+        
+        _uncompressed_size = strings[0];
+    }
+    /////////////////////////////////////////////////////////////////////////////
+
```
Comment (danielweck) [inline ePub3/ePub/encryption.cpp]: Note that this addresses this issue:
https://github.com/readium/readium-sdk/issues/162#issuecomment-77366075

**Comment by drminside** [comment]
I have tried to merge ```sdkForLcp``` into ``` develop ```, which has 1 conflict in the RDContainer.h.
So I send a pull request with #219 which is conflict resolved.

**Pull request merged**